src/App.css

```diff
@@ -250,16 +287,75 @@ nav {
   top: 0px;
   content: '';
   margin-right: 10px;
-  height: 32px;
-  width: 32px;
+  height: 4.2ch;
+  width: 4.2ch;
   border: 2px solid rgb(33, 33, 33);
 }
 
-.prompt {
-  display: grid;
-  grid-template-rows: auto auto;
-  grid-template-columns: auto;
-  grid-row-gap: 20px;
-  text-align: center;
+.list__item__delete-btn {
+  display: inline-flex;
+  align-items: center;
   justify-content: center;
+  flex-shrink: 0;
+  width: 4.5ch;
+  height: 4.5ch;
+  border-radius: none;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.list__item__delete-btn svg {
+  margin-bottom: 10px;
+}
+
```

Comment:
Something here seems not right when I am in my Desktop view using Firefox. The bins are not in one line (and I guess this is wanted). Also, when adding a very long item, the view seems a bit off.
<img width="353" alt="Screenshot 2021-05-13 at 20 39 13" src="https://user-images.githubusercontent.com/54622834/118208886-2a3b7600-b42d-11eb-90a5-2ab9cf124fc2.png">